ocs/ocp.py

```diff
@@ -65,21 +63,31 @@ def exec_oc_cmd(self, command):
 
         oc_cmd += command
         out = run_cmd(cmd=oc_cmd)
-        return munchify(yaml.safe_load(out))
+        # Removing any part of output which is out of the yaml/json
```

Comment:
get() has out_yaml_format
does this change mean it's not relevant anymore?

Reply:
it is relevant.
I missed something?

Reply:
exec_oc_cmd() will return a dict instead of munch, just want to make sure it doesn't affect the usage of get()

Reply:
I changed everything to use .get() from the dict  and also verified 4 test cases